modules/config/default/+evil-bindings.el

```diff
@@ -514,8 +515,9 @@
        :desc "Org agenda"                   "a" #'org-agenda
        (:when (featurep! :tools biblio)
         :desc "Bibliographic entries"        "b"
-        (cond ((featurep! :completion ivy)   #'ivy-bibtex)
-              ((featurep! :completion helm)  #'helm-bibtex)))
+        (cond ((featurep! :completion ivy)       #'ivy-bibtex)
+              ((featurep! :completion helm)      #'helm-bibtex)
+              ((featurep! :completion selectrum) #'bibtex-actions-open-entry)))
```

Comment:
On the commit message you ask if this is temporary, and yeah; @Emiller88 was trying to figure out how to implement my suggestion to have a prefix map for biblio; see here https://github.com/hlissner/doom-emacs/pull/4664#issuecomment-829335099.

I think ultimately we want bindings for each of the `bibtex-actions` commands, or at least the most important ones.

Reply:
yeah i saw that comment, I tried to figure out something similar, but it would be a shame to manually map everything rather than just map `bibtex-actions-map`. I'm sure there is some way to do it, if no one can answer on the discord we can leave it as a review item for henrik, he'd probably know.

Reply:
> it would be a shame to manually map everything rather than just map `bibtex-actions-map`.

Definitely.

That embark issue I submitted was inspired by me trying to figure out how to write a map that would work with doom and with embark, and would include the descriptions, rather than the command names.

I couldn't figure it out in the end, which is why the map currently has no descriptions.